<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateManualDotMatchingForm</class>
 <widget class="QDialog" name="CreateManualDotMatchingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Manual DotMatching</string>
  </property>
  <widget class="QLineEdit" name="EditLibID">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditLibName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>420</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyDotMatching.qrc">
     <normaloff>:/Qtres/Cancel.png</normaloff>:/Qtres/Cancel.png</iconset>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetLibList">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>201</width>
     <height>141</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="ButtonOK">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyDotMatching.qrc">
     <normaloff>:/Qtres/OK.png</normaloff>:/Qtres/OK.png</iconset>
   </property>
  </widget>
  <widget class="QFrame" name="frameLibFolder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_43">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>探
索</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>1</number>
    </property>
   </widget>
   <widget class="QFrame" name="frame_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label_44">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>20</width>
       <height>61</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>輝
度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QSpinBox" name="EditSelfSearch">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>ブロック単体での探索ドット数</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>371</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;検&lt;/p&gt;&lt;p&gt;出&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>8</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>明側</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>28</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>OK輝度範囲(%)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditOKDot">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>48</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>OKドット数</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditBrightWidthL">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>28</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditBrightWidthH">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>28</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>8</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>暗側</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_41">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>補
正</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>1</number>
    </property>
   </widget>
   <widget class="QFrame" name="frame_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label_42">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>20</width>
       <height>61</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>輝
度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QSpinBox" name="EditAdjustBlack">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditAdjustWhite">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>明側　最大補正輝度量</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>暗側　最大補正輝度量</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="PropertyDotMatching.qrc"/>
 </resources>
 <connections/>
</ui>
